File: analyzers/its/sources/ManuallyAddedNoncompliantIssues.VB/IntentionalFindings/S6561.vb
Imports System

Friend Class S6561
    Public Sub Benchmark()
        Dim start = Date.Now
        MethodToBeBenchmarked()
        Console.WriteLine($"{(Date.Now - start).TotalMilliseconds} ms") ' Noncompliant (S6561)
    End Sub
    Private Sub MethodToBeBenchmarked()
    End Sub
End Class

File: analyzers/its/sources/ManuallyAddedNoncompliantIssues.CS/IntentionalFindings/S6561.cs
using System;

namespace IntentionalFindings
{
    internal class S6561
    {
        public void Benchmark()
        {
            var start = DateTime.Now;
            MethodToBeBenchmarked();
            Console.WriteLine($"{(DateTime.Now - start).TotalMilliseconds} ms"); // Noncompliant (S6561)
        }

        void MethodToBeBenchmarked() { }
    }
}
